File: pylabs/lab1.py
import math
choice = input("Выберете функцию:1-G,2-F,3-Y " )

if choice =="1":
    a = float(input())
    x = float(input())
    q = (4*(-4*(a**2)+a*x+5*(x**2)))
    w = (-20*(a**2)+28*a*x+3*(x**2))
    G = q/w
    if ( w == 0) :
        print("Ошибка")
    else:
        print("G = %.2f "%G)
elif choice =="2" :
    a=float(input())
    x=float(input()) 
    F=(math.atan(24*(a**2)-25*a*x+6*(x**2)))
    print("F = %.2f"%F )
elif choice == "3":
    a=float(input())
    x=float(input())
    Y = math.log(2*(a**2)-7*a*x+6*(x**2)+1)
    print("Y = %.2f "% Y)
else:
    print("Ошибка ввода")

File: clabs/lab1.c
#include <stdio.h>
#include <stdio.h>
#include <math.h>
int main(void)
{
float a,x,g,f,y;
printf("Vvedite x: ");
scanf ("%f", &x);
printf("Vvedite a ");
scanf ("%f", &a);
g=4*(-4*a*a+a*x+5*x*x)/(-20*a*a+28*a*x+3*x*x);
printf("g=%f\n\n", g);

printf("Vvedite x ");
scanf ("%f", &x);
printf("Vvedite a ");
scanf ("%f", &a);
f=atan(24*a*a-25*a*x+6*x*x);
printf("f=%f\n\n", f);

printf("Vvedite x ");
scanf ("%f", &x);
printf("Vvedite a ");
scanf ("%f", &a);
y=log(2*a*a-7*a*x+6*x*x+1);
printf("y=%f\n\n", y);
}
